### PR TITLE
Update readme instructions

### PR DESCRIPTION
**Pull request by gustaveen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,92 @@
...
+#### Install Yarn
+
+1. Install `yarn` [https://classic.yarnpkg.com/en/docs/install/#mac-stable](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+
+#### Install yvm
+
+`yvm` is a neat yarn version manager just like `nvm` is for node.
+
+1. Install `yvm` [https://yvm.js.org/docs/overview#installation](https://yvm.js.org/docs/overview#installation)
+2. Run any yarn command and watch it automagically use the correct version of yarn. (Version specified in `package.json` or `.yvmrc`)
```
Comment (palmenhq) [inline README.md:30]: I'd like to challenge whether we should have this. Instead we should link to the [web contributing guidelines](https://github.com/HedvigInsurance/tech-stuff/blob/master/web/contributing.md) don't you think?

**Review by palmenhq — COMMENTED** [review]
Really nice!

File: README.md (hunk 1 above)
Comment (gustaveen) [reply README.md:30]: true, might be a bit overkill to repeat these steps in all repos. but we can add a link to the contributing guidelines maybe? @palmenhq

Comment (palmenhq) [reply README.md:30]: Yes this i think is a good idea!

**Review by palmenhq — DISMISSED** [review]

**Review by palmenhq — APPROVED** [review]

**Pull request merged**